src/extension/request_abort.c

```diff
@@ -178,7 +178,8 @@ void dd_set_redirect_code_and_location(int code, zend_string *nullable location)
 void dd_request_abort_redirect()
 {
     if (_redirection_location == NULL || ZSTR_LEN(_redirection_location) == 0) {
-        mlog(dd_log_warning, "Failing to redirect: No location set");
+        _response_code = DEFAULT_BLOCKING_RESPONSE_CODE;
```

Comment:
Shouldn't you revert both the code and type?

Reply:
Type is not change during the lifetime of a redirection. However, I would see ok if I create a method called `set_defaults` where it gets everything back to defaults if you are ok with it

Reply:
Type doesn't change during the redirection, but if we block, we never revert the type to the default value so it could be anything.

Reply:
I have done an alternative implementation where statics of blocking and redirection are not shared